examples/js/exporters/GLTFExporter.js

```diff
@@ -1922,7 +1922,7 @@ THREE.GLTFExporter.prototype = {
 
 			if ( scene.userData && Object.keys( scene.userData ).length > 0 ) {
 
-				gltfScene.extras = serializeUserData( scene );
+				gltfScene.extras = serializeUserData( scene, gltfScene );
```

Comment:
Hm, I think something else is wrong here. Can this whole `if` block be removed? A few lines later on we have `serializeUserData( scene, gltfScene );` which should do everything that is needed. `serializeUserData` doesn't return anything.

Reply:
Hmm, yeah I might have gone too fast on this one. I was getting a warning in the console in that particular call, and I didn't see the second call to serializeUserData with scene. Weird, I'm pretty sure I didn't have extras on my scene on the resulting GLTF. But again, I might have looked too quick...

The code I "fixed" must be leftover code from an old version.

Reply:
Yeah, I think we must have missed this when refactoring the userData writing. Thanks!